e2e/svc_mn_1786387615.py

```
@@ -0,0 +1,19 @@
+import hashlib
+import sqlite3
+
+
+def find_user(conn: sqlite3.Connection, username: str):
+    # SQL injection: the username is concatenated straight into the statement.
+    cursor = conn.cursor()
+    cursor.execute("SELECT id, email FROM users WHERE name = '" + username + "'")
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)